op-program/host/config/config.go

```diff
@@ -104,6 +105,7 @@ func (c *Config) Check() error {
 }
 
 func (c *Config) FetchingEnabled() bool {
+	// TODO: Include Beacon URL once cancun is active on all chains we fault prove.
```

Comment:
Probably make the beacon Url a required field if Cancun is scheduled for the chain in use.